ccmlib/scylla_docker_cluster.py

```diff
@@ -119,16 +130,23 @@ def create_docker(self):
             if 'alternator_https_port' in scylla_yaml:
                 ports += f" -v {scylla_yaml['alternator_https_port']}"
 
-            mount_points = []
-            for directory in ['data', 'commitlogs', 'hints', 'view_hints', 'saved_caches']:
-                mount_points.append(
-                    f'-v {os.path.join(self.get_path(),directory)}:{os.path.join(self.base_data_path, directory)}')
+            mount_points = ' '.join([
+                f'-v {os.path.join(self.get_path(),directory)}:{os.path.join(self.base_data_path, directory)}'
+                for directory in self.share_directories])
 
-            res = run(['bash', '-c', f"docker run {ports} -v {self.local_yaml_path}/scylla.yaml:/etc/scylla/scylla.yaml "
-                                     f"{' '.join(mount_points)} --name {self.docker_name} -v /tmp:/tmp "
-                                     f"-d {self.cluster.docker_image} --smp 1 {seeds}"], stdout=PIPE, stderr=PIPE)
+            res = run(['bash', '-c',
+                       f"docker run {ports} -v {self.local_yaml_path}/scylla.yaml:/etc/scylla/scylla.yaml"
+                       f" {mount_points} -v /tmp:/tmp --name {self.docker_name}"
+                       f" -d {self.cluster.docker_image} --smp 1 {seeds}"], stdout=PIPE, stderr=PIPE)
             self.pid = res.stdout.decode('utf-8').strip()
-
+            if self.cluster.snitch:
+                cassandra_rackdc_name = "cassandra-rackdc.properties"
+                run(["sudo", "bash", "-c",
+                     f"docker exec {self.pid}"
+                     f" cp {os.path.join(self.base_data_path, 'conf', cassandra_rackdc_name)}"
+                     f" {os.path.join('etc', 'scylla', cassandra_rackdc_name)}"])
+                run(['bash', '-c', f"docker exec {self.pid} supervisorctl restart scylla"],
```

Comment:
don't restart scylla here, handle cassandra-rackdc.properties exactly as we are doing scylla.yaml
copy it out of the docker, and mount the exact file into place

restarting scylla with different configuration might take time, and some wrong data might be create and cached already.

Reply:
this is handled in
https://github.com/scylladb/scylla-ccm/commit/3f3e3f80722e84751f99a985fee2265744cda9d4

Reply:
Thanks!